### PR TITLE
add run recent and go to cwd to command decoration menu

### LINKED ISSUE #170979: Explore having run recent command and go to recent directory in the shell integration menu
Here:

image

This will help with discovering the keybindings.

Context: https://github.com/microsoft/vscode/issues/170634#issuecomment-1376001378

### PR DESCRIPTION
**Pull request by meganrogge**
Screenshot 2023-01-10 at 2 04 17 PM

fix #170979

### CODE REVIEW

**Review by Tyriar — COMMENTED** [review]
Let's hold off on this for now and bring up along with https://github.com/microsoft/vscode/pull/170993 in the UX sync tomorrow

**Comment by meganrogge** [comment]
I think we decided to go with this change, correct @Tyriar ? I think you mentioned removing the category, but IMO it feels more organized to keep the related ones grouped.

**Review by Tyriar — APPROVED** [review]
Yeah the only thing I'm not sure about is the first separator as it's a lot of categories, I think this is better as then the categories are:

- Command-related actions
- Terminal related actions
- Help

image

It's also obvious that one deals with input and one with output because of Command/Output in the label. I'll let you decide whether we go with that or not, we can always change later.

**Pull request merged**
